.chglog/config.yml

```diff
@@ -18,10 +18,21 @@ options:
       perf: Performance Improvements
       refactor: Code Refactoring
   header:
-    pattern: "^(\\w*)\\:\\s(.*)$"
+    pattern: "^(?:(\\w*)|(?:\\[(.*)\\])?)\\:\\s(.*)$"
     pattern_maps:
       - Type
+      - JiraIssueId
       - Subject
   notes:
     keywords:
       - BREAKING CHANGE
+  jira:
+    #info:
+      #username: <jira-username>
+      #token: <jira-password>
+      #url: <https://jira.com>
+    issue:
+      type_maps:
+        Task: fix
+        Story: feat
+      description_pattern: "<changelog>(.*)</changelog>"
```

Comment:
I am happy to hear that you worked out the issue with the `pattern` on line 21.

The issue here is that we do not want to modify the config file that is used by the `git-chglog` project within this PR. The `jira` block should not be required for the tool to operate correctly and this file is what is used to generate the `CHANGELOG.md` within the project repo.

Please revert these changes to this file.

I think that the `pattern` regex should be added as a `template` for within the `Formats` (see: https://github.com/git-chglog/git-chglog/blob/master/cmd/git-chglog/variables.go#L111-L119) that are provided during the `--init` command. This will allow a User to choose a Jira specific configuration during initialization.

Reply:
OK, reverted. I won't update `variables.go` in this PR. If you really need, I can do it in a new PR.

But for now, I want this PR to be merged first. Then I'll going to submit a new PR to add a new query syntax: "tag1->tag2", details will be in the new PR.

Reply:
Thank you. A follow up to PR that updates the variables file and, as a result, the init process would be a nice way to expose users to this feature. 

I am cool with it as follow up work.